src/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataAggregator.cs

```diff
@@ -64,14 +64,10 @@ public MetadataAggregator(MetadataReader baseReader, IReadOnlyList<MetadataReade
                 }
 
                 CalculateBaseCounts(baseReader, out baseTableRowCounts, out baseHeapSizes);
+                Debug.Assert(baseTableRowCounts != null);
```

Comment:
The person who ran the analyzer guessed that the exception might have been intended to be placed here.